add links for active groups, mark others as inactive

pdurbin opened the pull request:
Here's a preview:

![Screenshot 2023-12-12 at 9 35 07 AM](https://github.com/gdcc/dataversecommunity.global/assets/21006/cc078fb2-8832-4abc-8961-3c49e816b5dc)

Of the four active working groups, I believe only the container group has a website. The other groups should probably work on having something better to link to (I linked to what I could).

I wrote "inactive" next to the rest of the groups but we could also just delete them? Or group them together under and "inactive" heading at the bottom?

cmbz commented:
@pdurbin The Big Data working group is active. @sbarbosadataverse leads the effort.

pdurbin commented:
@cmbz oh! Great! Do you or @sbarbosadataverse have a suggestion of what to link to?

sbarbosadataverse commented:
wg_largedatasupport

pdurbin commented:
@sbarbosadataverse thanks, in 10c4d97 I added a link for the Big Data group and removed "inactive". We can change it from "Big Data" to "Large Data Support" if you'd like.

JR-1991 commented:
PyDataverse IG will begin on January 24. I am happy to set up a website. Can I use a template or create something on my own, @pdurbin?

pdurbin commented:
@JR-1991 you may have noticed that in this PR, the Containerization Working Group links to its website at https://ct.gdcc.io

If you scroll down, you'll see the source is at https://github.com/gdcc/ct.gdcc.io

You're very welcome to use that repo as a starting point if you wish! @poikilotherm controls DNS for gdcc.io but I'm sure if we buy him a Coke or two he'll set up an entry for us. 😄

All that said, there's no requirement for a website. It could just be a Google doc or anything that will be kept up to date with the next meeting time, a Zoom link, etc.

Oh and to everyone, we can add all events to the [Dataverse Community Calendar](https://calendar.google.com/calendar/embed?src=c_udn4tonm401kgjjre4jl4ja0cs%40group.calendar.google.com&ctz=America%2FNew_York), which you can find linked from https://dataverse.org/events . I could also update this pull request to link to it and say something like "most working groups include their meetings on the community calendar". @philippconzett let me know what you think!

philippconzett commented:
Thanks, @pdurbin, this looks all good to me! Can you merge this PR, or do you need any input from me?

pdurbin commented:
@philippconzett merged! https://www.gdcc.io/working-groups.html has been updated.

@JR-1991 nothing has added for pyDataverse yet. Can you please make a PR when you're ready?

@kmika11 if there's a better link for Sensitive Data, please create a PR to fix it.

@sbarbosadataverse and @ekraffmiller same for Big Data and UI Components.

Thanks!

Here's how the page looks post-merge:

![Screenshot 2023-12-15 at 9 22 36 AM](https://github.com/gdcc/dataversecommunity.global/assets/21006/fc2c52b3-ef51-4db7-92e4-33a5a3c7099f)

Rather than maintaining two pages, I updated https://dataverse.org/working-groups to point to the GDCC page:

![Screenshot 2023-12-15 at 9 25 39 AM](https://github.com/gdcc/dataversecommunity.global/assets/21006/84339ed6-bb59-417d-961f-dea5a1caa1f7)

JR-1991 commented:
@pdurbin thanks for the notice. Added the link to the repo in PR #2
